packages/npm/http-client-csharp/emitter/src/emitter.ts

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Why are we putting ts into the .NET repo?

Reply:
The idea is to have the emitter & generator within the same place as all the SDKs, so that SDK generation will just consume the local package instead of consume another package from another repo.
And also, we always regen all the SDKs while update emitter or generator in `autorest.csharp`.
With them in the same place, it is easier to regen them in a CI step of the same repo.

Reply:
Is it a concern that this plugin inherits from a plugin in another repo?  How will updates be applied across repos?  If a change is made to the base type in the other repo that causes the AzureClientPlugin to produce invalid code for Azure clients, at what point would we catch those problems?

Reply:
@m-nash and I chatted about this, and he is working on a plan for where we want to ultimately put this. I'm generally not a fan of putting tools and releasing them from the sdk repos as the primary goal is about shipping sdks not other things. My recommendation to @m-nash is to put this in another repo, perhaps the azure-sdk-tools repo, or if it must remain in this repo it needs to be hidden away somewhere under the eng folder and visible to the normal user coming into the repo.

Reply:
> Is it a concern that this plugin inherits from a plugin in another repo? How will updates be applied across repos? If a change is made to the base type in the other repo that causes the AzureClientPlugin to produce invalid code for Azure clients, at what point would we catch those problems?

`AzureClientPlugin` inherits from a fixed version of [ClientModelPlugin](https://github.com/microsoft/typespec/blob/main/packages/http-client-csharp/generator/Microsoft.Generator.CSharp.ClientModel/src/ClientModelPlugin.cs), we should be able to detect the unexpected changes to SDK generation during the version bump same as we are doing with TCGC and `autorest.csharp` right now.